FusionIIIT/applications/visitor_hostel/models.py

```diff
@@ -100,21 +99,20 @@ class BookingDetail(models.Model):
     number_of_rooms_alloted =  models.IntegerField(default=1,null=True,blank=True)
     booking_date = models.DateField(auto_now_add=False, auto_now=False, default=timezone.now)
     bill_to_be_settled_by = models.CharField(max_length=15, choices=BILL_TO_BE_SETTLED_BY ,default ="Intender")
-    
+
     def __str__(self):
         return '%s ----> %s - %s id is %s and category is %s' % (self.id, self.visitor, self.status, self.id, self.visitor_category)
 
 
 class MealRecord(models.Model):
     booking = models.ForeignKey(BookingDetail, on_delete=models.CASCADE)
     visitor = models.ForeignKey(VisitorDetail, on_delete=models.CASCADE)
-    room = models.ForeignKey(RoomDetail, on_delete=models.CASCADE, default=0)
     meal_date = models.DateField()
-    morning_tea = models.BooleanField(default=False)
-    eve_tea = models.BooleanField(default=False)
-    breakfast = models.BooleanField(default=False)
-    lunch = models.BooleanField(default=False)
-    dinner = models.BooleanField(default=False)
+    morning_tea = models.IntegerField(default=0)
+    eve_tea = models.IntegerField(default=0)
+    breakfast = models.IntegerField(default=0)
+    lunch = models.IntegerField(default=0)
+    dinner = models.IntegerField(default=0)
     persons=models.IntegerField(default=0)

```

Comment:
What's the effect on previous entries after changing FieldType?

Reply:
We are dopping and recreating table